248.breaking.rst

```diff
@@ -0,0 +1 @@
+Drop support for python 3.7
```

Comment:
Noticed these are placed outside of the `newsfragments` dir.

Reply:
oof! good catch, thank you!